futures-util/src/async_await/select.rs

```diff
@@ -10,13 +10,23 @@ macro_rules! select {
     )*) => { {
         $(
             $crate::async_await::assert_unpin(&$name);
-            let mut $name = $crate::future::maybe_done(&mut $name);
-            let mut $name = $crate::core_reexport::mem::PinMut::new(&mut $name);
         )*
+        #[allow(bad_style)]
+        struct __priv<$($name,)*> {
```

Comment:
The name should be capitalized

Reply:
...why?

Reply:
because all structs have a name that starts with an upper case letter by convention. `__Priv` would suggest that it's a type, not a binding